vue/components/ui/molecules/table/table.vue

```diff
@@ -196,6 +205,11 @@
     width: 20px;
 }
 
+.table[style="text-align: left;"] .table-column > span::before {

```

Comment:
Instead of this, add a class (`table-align-left`) that represents the alignment and use it here